### ISSUE TITLE
Dot Voting: remove exponent notation in Participation

### ISSUE THREAD

**Issue by e18r** [AutarkLabs/open-enterprise#1704]
When displaying a very small value, instead of using exponent notation,
![Screenshot from 2019-12-03 13-06-54](https://user-images.githubusercontent.com/16065447/70077012-f6201c80-15cd-11ea-886f-6fb4980d3005.png)

Use two decimals. In this case, it would be "0.00%".

**Comment by ottodevs** [comment]
I don't think this needs the "needs design" label, it seems more like a cosmetic bug, maybe we should apply something like `Math.min(actualValue, 0.01)` just for display purposes, even if internally we allow smaller percentages.

**Comment by javieralaves** [comment]
@ottodevs Agreed, maybe instead of "0.00%", show ">0.00%"? Just to make it clear that's it's not exactly zero, but a very small amount. Re: pipeline, it's just a practice that Yalda and I have been doing lately to move issues to `Icebox` when they're not necessarily time-sensitive to upcoming sprints. The idea is to get as many issues out of `New issues` as possible! Does that make sense?
